legion/sdk/legion/sdk/clients/deployment.py

```diff
@@ -56,11 +56,11 @@ def from_json(md: typing.Dict[str, str]) -> 'ModelDeployment':
         return ModelDeployment(
             name=md.get("name"),
             image=md_spec.get('image', ''),
-            resources=md_spec.get('resources', ''),
-            annotations=md_spec.get('annotations', []),
+            resources=md_spec.get('resources', {}),
+            annotations=md_spec.get('annotations', {}),
             replicas=md_spec.get('replicas', ''),
-            liveness_probe_initial_delay=md_spec.get('livenessProbeInitialDelay', ''),
-            readiness_probeInitial_delay=md_spec.get('readinessProbeInitialDelay', ''),
+            liveness_probe_initial_delay=md_spec.get('livenessProbeInitialDelay'),
+            readiness_probeInitial_delay=md_spec.get('readinessProbeInitialDelay'),
```

Comment:
typo in readiness_pro**beIn**itial_delay, please fix

Reply:
Fixed